sql_extensions/Dockerfile

```diff
@@ -6,5 +6,6 @@ WORKDIR /
 
 COPY sql_extensions/00000_init.sql .
 COPY sql_extensions/apply-sql-extensions.sh .
+COPY sql_extensions/migrations/ ./migrations/
```

Comment:
Is this the right way to do this?

Reply:
Yes. This was an error that is now fixed if you rebase using main.